README.md

```diff
@@ -1 +1,89 @@
-# lodge
\ No newline at end of file
+# lodge
+
+A simple log lib used by Neoway's ML Platform team.
+
+## Features
+
+* Easy to use
+* Env var configurations
+* Structured production log ready!
+* Options to customize fields
+* Set log level by module
+* Based on stdlib
+
+## Getting Started
+
+```python
+from lodge import log
+
+log.info("Is that simple?")
+# {"message": "Is that simple?", "timestamp": "2021-03-21 14:26:54,838", "level": "INFO"}
+```
+
+### Global level with env var config
+
+```python
+# export LOG_LEVEL=ERROR
+
+from lodge import log
+
+log.info("This will not appear")
+log.error("Oh no")
+# {"message": "Oh no", "timestamp": "2021-03-21 14:26:54,839", "level": "ERROR"}
+```
+
+### Module level with env var config
+
+```python
+# export PACKAGE_MODULE1_LOG_LEVEL=ERROR
+# export PACKAGE_MODULE2_LOG_LEVEL=INFO
+
+# package/module1.py
+from lodge import log
+
+log.info("Module 1 info")
+...
+log.error("Module 1 error")
+# {"message": "Module 1 error", "timestamp": "2021-03-21 14:26:54,839", "level": "ERROR"}
+
+# package/module2.py
+from lodge import log
+
+log.info("Module 2 info")
+# {"message": "Module 2 info", "timestamp": "2021-03-21 14:26:54,839", "level": "INFO"}
+...
+log.error("Module 2 error")
+# {"message": "Module 2 error", "timestamp": "2021-03-21 14:26:54,839", "level": "ERROR"}
+```
+
+### Change format to easier read on development
+```python
+# export LOG_ENV=DEV
+
+# package/module1.py
+from lodge import log
+
+log.info("a message")
+
+# 2021-03-21 14:34:47,273 | INFO | package.module1 | a message
+```
+
+### Add extra fields
```

Comment:
@mfrata is it possible to use an extra field filled at runtime? Example:

```python
$ export LOG_EXTRA_FIELDS='{"client_id": "%client_id"}'

>>> from lodge import log
>>> log.error("some error", client_id=1)
```

or does it make sense?

Reply:
I thought on implement it. But this first release was meant to be simple. I'm going to turn this into an issue to be addressed later.